README.md

```diff
@@ -34,14 +34,14 @@ As such, we propose the adoption of a syntax to simplify this common pattern:
 
 ```js
 function * g() {
-  using const handle = acquireFileHandle(); // block-scoped critical resource
+  using handle = acquireFileHandle(); // block-scoped critical resource
 
   // or, if `handle` binding is unused:
-  using const void = acquireFileHandle(); // block-scoped critical resource
+  using void = acquireFileHandle(); // block-scoped critical resource
```

Comment:
```suggestion
  using acquireFileHandle(); // block-scoped critical resource
```

Is it possible?

Reply:
Not really. `` `using` AssignmentExpression `` would conflict with `using(x)` (which is already legal JavaScript). The other point of `using void = ` was that it could be used in a _BindingList_:

```js
using const x = ...,
            void = ...,
            y = ...;
```

Reply:
> `` `using` AssignmentExpression `` would conflict with `using(x)` (which is already legal JavaScript). 

That is, unless we did something like `` `using` [lookahead != `(`] LeftHandSideExpression `;` ``, however I could see potential uses like the following that wouldn't work with the above:

```js
using void = resource ?? getResource();
```
or
```js
using void = (resource ??= getResource());
```